drivers/input/tsc2007.c

```diff
@@ -1234,7 +1234,7 @@ int tsc2007_register(FAR struct i2c_master_s *dev,
    * have priority inheritance enabled.
    */
```

Comment:
/* The event dev semaphore is used for lock, hence, should have priority inheritance enabled. */